### PR TITLE
Nojira | @ericbakenhus | Fix mobile filter text color

### PR DESCRIPTION
**Pull request by ericbakenhus**
# READY FOR REVIEW

# Summary
- Dangit

# Review By (Date)
- Dangit

# Criticality
- Dangit

# Review Tasks

## Setup tasks and/or behavior to test

1. Dangit

### CODE REVIEW

File: src/components/events-discovery/components/Facets/MobileFacetList.jsx [hunk 1]
```diff
@@ -25,7 +25,7 @@ export const MobileFacetList = ({ attribute }) => {
           {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
           <label
             htmlFor={item.label}
-            className="su-group su-flex su-items-center su-font-sans su-pl-16 su-text-cardinal-red su-w-full su-transition-colors hover:su-bg-cardinal-red-light hover:su-text-white"
+            className="su-group su-flex su-items-center su-font-sans su-pl-16 su-text-black su-w-full su-transition-colors hover:su-bg-cardinal-red-light hover:su-text-white"
```
Comment (ericbakenhus) [inline src/components/events-discovery/components/Facets/MobileFacetList.jsx:28]: Dangit.

File: src/components/events-discovery/components/Facets/MobileFacetList.jsx [hunk 2]
```diff
@@ -38,7 +38,10 @@ export const MobileFacetList = ({ attribute }) => {
             <span className="su-font-sans su-flex su-w-full su-text-18 su-pl-16 su-pr-26 su-py-14 su-cursor-pointer group-hover:su-underline peer-focus:su-underline">
               <span className="su-grow">
                 {item.label}
-                <span className="su-font-sans"> ({item.count})</span>
+                <span className="su-font-sans su-text-cardinal-red">
+                  {' '}
+                  ({item.count})
+                </span>
```
Comment (ericbakenhus) [inline src/components/events-discovery/components/Facets/MobileFacetList.jsx:44]: Keep this red, dangit.

File: src/components/events-discovery/components/Filters/MobileParentFilter.jsx [hunk 3]
```diff
@@ -17,7 +17,7 @@ export const MobileParentFilter = ({
   <>
     <button
       type="button"
-      className="su-group su-flex su-items-center su-w-full su-rounded-none su-p-16 su-border-b hover:su-bg-cardinal-red-light hover:su-text-white focus-visible:su-bg-cardinal-red-light focus-visible:su-text-white"
+      className="su-group su-flex su-items-center su-w-full su-rounded-none su-p-16 su-border-b su-text-black hover:su-bg-cardinal-red-light hover:su-text-white focus-visible:su-bg-cardinal-red-light focus-visible:su-text-white"
```
Comment (ericbakenhus) [inline src/components/events-discovery/components/Filters/MobileParentFilter.jsx:20]: Just to make dang sure this stays black text.

**Review by sherakama — COMMENTED** [review]
Ah. Dang.

**Pull request merged**